contrib/babelfishpg_common/src/sqlvariant.c

```diff
@@ -2306,6 +2308,27 @@ bytea *convertIntToSQLVariantByteA(int ret) {
 	return result;
 }
 
+bytea *convertBinaryToSQLVariantByteA(int64_t ret)
```

Comment:
It is tough to understand what we are doing inside this API for the person looking this first time. 
I would suggest you to include this logic in tdscollationproperty_helper only and add the comment over there to explain what we are doing.

Reply:
Implemented the changes